adam-cli/src/test/scala/FlagStatTest.scala

```diff
@@ -0,0 +1,136 @@
+package org.bdgenomics.adam.cli
+import org.apache.spark.rdd.RDD
+
+import org.bdgenomics.adam.projections.{AlignmentRecordField, Projection}
+import org.bdgenomics.adam.rdd.ADAMContext._
+import org.bdgenomics.adam.rdd.read.{DuplicateMetrics, FlagStatMetrics}
+import org.bdgenomics.adam.util.SparkFunSuite
+import org.bdgenomics.formats.avro.AlignmentRecord
+import org.bdgenomics.adam.rdd.read.FlagStat._
+
+class flagstat_test extends SparkFunSuite with java.io.Serializable   {
+
+  sparkTest("Standard FlagStat test"){
+
+    val inputpath = ClassLoader.getSystemClassLoader.getResource("features/NA12878.adam").getFile
```

Comment:
This is throwing an NPE which most likely means that `getResource` is returning `null`. Does the `features/NA12878.adam` file exist?